blazor/mention/getting-started.md

```diff
@@ -305,6 +305,54 @@ For Blazor WebAssembly App, refer script in the `<head>` of the **~/index.html**
 
 ![Blazor Mention Component](images/blazor-mention.png)
 
+## Mention target
+
+The `Target` property of the Mention component can be used to specify the element that the mention element should be appended to, and this can be useful in cases where you want to display the mention element in a specific location on the page.
```

Comment:
specify the element , 

also mention the css selector element ID and class selector

Reply:
The `Target` property of the Mention component allows you to specify an element on the page to which the mention element should be attached. This can be useful when you want to display the mention element in a specific location on the page. For example, you might use the `Target` property to attach the mention element to a specific div element or to a specific input field or to a specific textarea field. To specify the target element, you can pass a CSS selector, a DOM element.